client/blocks/signup-form/index.jsx

```diff
@@ -629,10 +628,6 @@ class SignupForm extends Component {
 		);
 	}
 
-	isJetpackSocialABTest() {
-		return this.isJetpack() && abtest( 'jetpackSignupGoogleTop' ) === 'top';
```

Comment:
With this removal, we don't need `isJetpack()` as a separate method too (it was also introduced in #24531).

Reply:
well, yeah, but since it's still being used, even if it's in just one place... it adds a little bit of readability :)